src/frontend/src/lib/components/core/Menu.svelte

```diff
@@ -68,6 +72,11 @@
 
 		<Hr />
 
-		<SignOut on:icLogoutTriggered={() => (visible = false)} />
+		<AboutWhat asMenuItem on:icOpenAboutModal={hidePopover} />
```

Comment:
`on:click` is not used - i.e. the modal does not appear / work. Is that expected?

Reply:
I was going to do it in another PR, but it does not make sense... I included the modals here.